godot-codegen/src/class_generator.rs

```diff
@@ -490,11 +498,14 @@ fn is_method_excluded(method: &ClassMethod, #[allow(unused_variables)] ctx: &mut
     }
     // -- end.
 
-    method.name.starts_with('_')
-        || method
-            .return_value
-            .as_ref()
-            .map_or(false, |ret| ret.type_.contains('*'))
+    if method.name.starts_with('_') && !is_virtual_impl {
+        return true;
+    }
```

Comment:
I removed the blanket ban on all method names starting with `_`. All virtual methods have a Godot name that starts with an underscore, so that would exclude all of the methods I intend to add with this MR. The pointer condition is still present (line 508). Looks like the Rust formatter just re-indented it a bit, which is why it shows as a diff.

Reply:
Oh, I see. This is where semantic diff like [difftastic](https://github.com/Wilfred/difftastic) might come in really handy 😎

All good then, thanks for explanation!